.github/workflows/test_behavior_binding_go.yml

```diff
@@ -55,8 +55,8 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/checkout@v4
         with:
-            repository: "apache/opendal-go-services"
-            path: "tools"
+          repository: "apache/opendal-go-services"
+          path: "tools"
       - name: Setup Rust toolchain
```

Comment:
I think this step can be removed since feature flag is available

Reply:
```
      - uses: actions/checkout@v4
        with:
          repository: "apache/opendal-go-services"
          path: "tools"
```

Are you referring to this step?

Reply:
So `tools/.github/scripts/setup_features.py` is not needed too?